subiquity/core.py

```diff
@@ -182,6 +184,31 @@ def report_finish_event(self, name, description, status, level="INFO"):
         self.controllers.Reporting.report_finish_event(
             name, description, status, level)
 
+    def confirm_install(self):
+        self.install_confirmed = True
+        self.controllers.InstallProgress.confirmation.set()
+
+    def next_screen(self):
+        can_install = all(e.is_set() for e in self.base_model.install_events)
+        if can_install and not self.install_confirmed:
+            if self.interactive():
+                from subiquity.ui.views.installprogress import (
+                    InstallConfirmation,
+                    )
+                self.ui.body.show_stretchy_overlay(
+                    InstallConfirmation(self.ui.body, self))
+            else:
+                answer = 'no'
+                if 'autoinstall' in self.kernel_cmdline:
+                    answer = 'yes'
+                while answer != 'yes':
+                    print("\n\nContinue with autoinstall? (yes|no)")
+                    answer = input()
```

Comment:
All of the above needs to be translated. Can we use existing translated strings from filesystem steps?

Reply:
Also i wonder if we can use `systemd-ask-password` with like all the agents that ask via console, wall, and allow people to ssh in to answer via `systemd-tty-ask-password-agent --query` (or like wrap that into `subiquity-continue-autoinstall`).

Reply:
Or like reuse any suitable string we can find in gnome/d-i to import translations.

Reply:
I've added _()s. Happy to make this cleverer but perhaps not in this branch? In general the thing is going to be people adding autoinstall to the kernel command line. Maybe I should add a hint about that.

Reply:
Should we add a boot menu item for it?

Reply:
Maybe hidden away in an advanced options menu or something. I think it would be confusing to make it too prominent.